lib/hyrax/hirmeos/client.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+class Hyrax::Hirmeos::Client
+  attr_accessor :username, :password, :metrics_base_url, :token_base_url, :translation_base_url
+
+  def initialize(username, password, metrics_base_url, token_base_url, translation_base_url)
+    @username = username
+    @password = password
+    @metrics_base_url = metrics_base_url
+    @token_base_url = token_base_url
+    @translation_base_url = translation_base_url
+  end
+
+  def post_work(work)
+    con = id_translation_connection
+    con.post('/works', work.to_json)
+  end
+
+  def get_work(uuid)
+    con = metrics_connection
+    con.get("/events?filter=work_uri:urn:uuid:#{uuid}") # This will need to be made configurable I think?
+  end
+
+  def request_token
+    response = Faraday.post(URI.join(token_base_url, 'tokens')), { email: username, password: password }.to_json
+    JSON.parse(response[0].body)['data'][0]['token']
```

Comment:
Do you think this would benefit from `dig`?  I'm a little afraid of chains like these being brittle but maybe it is not an issue if HIRMEOS always returns consistent json or errors (and errors just would bubble up here as expected for now).

Reply:
I tried to get digs to work but couldn't get it going. We can keep an eye on it